otter/integration/tests/test_pause.py

```diff
@@ -0,0 +1,187 @@
+        2. Creating a group with 1 min entity
+        3. Pause the group
+        4. Wait for server to finish building in mimic
+        5. Notice that group continues to think that server is building
+        """
+        mimic_nova = MimicNova(pool=self.helper.pool, test_case=self)
+        server_build_time = convergence_interval + 5
+        yield mimic_nova.sequenced_behaviors(
+            self.rcs,
+            criteria=[{"server_name": "pause-stops-convergence" + ".*"}],
+            behaviors=[
+                {"name": "build",
+                 "parameters": {"duration": server_build_time}}
+            ])
+        group, _ = self.helper.create_group(min_entities=1)
+        yield group.start(self.rcs, self)
+        one_building = ContainsDict({"pendingCapacity": Equals(1),
+                                     "activeCapacity": Equals(0),
+                                     "status": Equals("ACTIVE")})
+        yield self.helper.assert_group_state(group, one_building)
+        yield group.pause(self.rcs)
+        # Wait for server to build and few more convergence cycles after that
+        yield sleep(reactor,
+                    server_build_time + convergence_interval * 2)
+        # The group still thinks that server is building
+        yield self.helper.assert_group_state(group, one_building)
+
+    @inlineCallbacks
+    def test_pause_and_execute_policy(self):
+        """
+        Executing any policy of a paused group will result in 403
+        """
+        group, _ = self.helper.create_group()
+        yield group.start(self.rcs, self)
+        policy = ScalingPolicy(set_to=1, scaling_group=group)
+        yield policy.start(self.rcs, self)
+        yield group.pause(self.rcs)
+        yield policy.execute(self.rcs, [403])
+
+    @inlineCallbacks
+    def test_pause_and_create_policy(self):
+        """
+        Policy can be created on a paused group
+        """
+        group, _ = self.helper.create_group()
+        yield group.start(self.rcs, self)
+        yield group.pause(self.rcs)
+        policy = ScalingPolicy(set_to=1, scaling_group=group)
+        yield policy.start(self.rcs, self)
+        returnValue((group, policy))
+
+    @inlineCallbacks
+    def test_pause_and_create_execute_webhook(self):
+        """
+        Webhook can be created on a paused group but it cannot be executed
+        """
+        group, policy = yield self.test_pause_and_create_policy()
+        webhook = yield policy.create_webhook(self.rcs)
+        # execute webhook and wait for sometime for execution to take place
+        resp = yield treq.post(webhook.capurl, pool=self.helper.pool)
+        self.assertEqual(resp.code, 202)
+        yield treq.content(resp)
+        yield sleep(reactor, 2)
+        # The group has no impact
+        yield self.helper.assert_group_state(
+            group,
+            ContainsDict({"pendingCapacity": Equals(0),
+                          "activeCapacity": Equals(0)}))
```

Comment:
Should this also check `desiredCapacity` to assert that there's been no change?  I know that a change to `desiredCapacity` would be reflected in `pendingCapacity`, since the pending is now just (desired - active), but it might be nice to check all 3 because desired is the actual number that gets changed when a policy gets executed, and also if there's ever a bug in the pending/active/desired calculation this would catch it.  :)